src/storage/mvcc/txn.rs

```diff
@@ -458,6 +458,22 @@ impl<S: Snapshot> MvccTxn<S> {
                         pessimistic: true,
                     });
                 }
+                // A lock with larger min_commit_ts than current commit_ts can't be committed
+                if commit_ts < lock.min_commit_ts {
+                    error!(
```

Comment:
Not necessary to log with error

Reply:
Oh, sorry.
Is info sufficient?

Reply:
I think info is OK. It's normal that this case happens.